root.spec

```diff
@@ -2,7 +2,7 @@
 ## INITENV +PATH PYTHONPATH %{i}/lib
 ## INITENV SET ROOTSYS %{i}
 %define tag c99f334c8d6bd505f4f0e453dcc654a938020e10
-%define branch cms/v6-14-00-patches/94610d9
+%define branch cms/v6-14-00-patches/65ed49a
```

Comment:
this will not work. We do not have  cms/v6-14-00-patches/65ed49a branch in https://github.com/cms-sw/root/. Let me open a PR with the proper fix

Reply:
Thanks, closing this.